.changeset/breezy-dryers-thank.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/backend-tasks': patch
+---
+
+Instrument `backend-tasks` with some counters and histograms for duration
```

Comment:
Might wanna mention what the metrics are! This is cool new info. Could document them in the package readme too

Reply:
😉 https://github.com/backstage/backstage/issues/20155